.travis.yml

```diff
@@ -1,22 +1,30 @@
 services:
   - docker
 env:
-  matrix:
-    - ARCH="AMD64" BASE_IMAGE=alpine:3.12 OCAML_VERSION=4.12.0 UNISON_VERSION=2.51.3
-    - ARCH="AMD64" BASE_IMAGE=alpine:3.12 OCAML_VERSION=4.12.0 UNISON_VERSION=2.51.4_rc2
-    - ARCH="ARM64" BASE_IMAGE=arm64v8/alpine:3.12 OCAML_VERSION=4.12.0 UNISON_VERSION=2.51.3
-    - ARCH="ARM64" BASE_IMAGE=arm64v8/alpine:3.12 OCAML_VERSION=4.12.0 UNISON_VERSION=2.51.4_rc2
+  global:
+    - OCAML_VERSION=4.12.0
 jobs:
   include:
     - name: default
       script: docker build . -t eugenmayer/unison:latest
+    - env: BASE_IMAGE=alpine:3.12 UNISON_VERSION=2.51.3
+      arch: amd64
```

Comment:
is arch a travis specific arg and influences the arch it builds on?

Reply:
Yes, `arch` is a Travis CI arg. https://docs.travis-ci.com/user/multi-cpu-architectures/#multi-cpu-availaibility